### ISSUE TITLE
Problem with latex chapter style

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#147]
I want to personalize the style of the chapters modifying the conf.py. For example I add this piece of code:

latex_elements = {
    'fncychap' : '\usepackage[Bjornstrup]{fncychap}',
    }

When I build the latex pdf the style is not applied. This is because the resulting tex file includes the shphinx package after Bjornstrup package.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/147
- Originally reported by: Anonymous
- Originally created at: 2009-04-30T18:39:20.513

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-05-02 17:50:07+00:00_

But why is that a problem?

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-05-03 08:38:40+00:00_

Are you sure you're using the correct LaTeX syntax? I've just tested this, and it seems to work fine.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:06+00:00_

Removing version: 0.6 (automated comment)

**Comment by liminkai520520** [comment]
it seems to work fine

**Comment by jfbu** [comment]
I am reopening because indeed the `'fncychap'` key does not work as expected. With:

``` python
latex_elements = {

'fncychap' : '\\usepackage[Lenny]{fncychap}',
}
```

we obtain

![capture d ecran 2016-10-07 a 10 36 17](https://cloud.githubusercontent.com/assets/2589111/19183719/ef6ebcba-8c79-11e6-8ea7-7fc2f5cd09e7.png)

whereas the correct result should be

![capture d ecran 2016-10-07 a 10 37 27](https://cloud.githubusercontent.com/assets/2589111/19183742/13d43daa-8c7a-11e6-946d-e6ee3d9103ab.png)

The problem is with the code in `sphinx.sty` (mentioned in #1986) which is executed after the loading of package `fncychap` and tampers with the effect of its option `Lenny` (in this case). It seems the goal of this is to modify the `Bjarne` style, but the correct course of action should be for `sphinx.sty` to do `\usepackage[Bjarne]{fncychap}` and then customize it (for backwards compatibility) _except_ if user has made use of `'fancychap'` key.

**Comment by jfbu** [comment]
@shimizukawa @tk0miya Sphinx should not ship `fncychap` anymore, the shipped version is the latest one `2007/07/30 v1.34` and the upstream has not been updated since. If we do ship it then we should patch it, for example some of its options use `OT1` encoded TeX fonts which is very inappropriate for Sphinx use. If we did that we could incorporate the code from `sphinx.sty` directly in patched `fncychap` and there is the question whether `\py@HeaderFamily` should modify native behavior of style options of `fncychap`.

My own opinion is that we should not ship `fncychap` nor modify how its options work, despite that some are inadequate because they use `OT1` encoded TeX fonts, which is only ok for English, and also despite the fact that `fncychap` uses commands like `\rm` which do work in LaTeX but if user opts for a KOMA-script class (frequently used by German speakers) the `\rm`, `\bf` .... now are all declared invalid and raise compilation error ! (bad decision IMHO, LaTeX itself never declared these commands invalid).

It is possible for `sphinx.sty` to check if `fncychap` was loaded with option `Bjarne` and only then do its modification to the style as currently in `sphinx.sty`. This would fix the issue here.

**Comment by jfbu** [comment]
@shimizukawa @tk0miya I have tested the Sphinx PDF documentation using

``` python
'fncychap' : '\usepackage[Bjornstrup]{fncychap}',
```

in `doc/conf.py`, and result is not bad now. Perhaps we can do it for next release of Sphinx ?

![capture du 2016-10-11 15-28-36](https://cloud.githubusercontent.com/assets/2589111/19272196/f15034a4-8fc7-11e6-98bb-5f00f9c70c60.png)
